src/presentation/ui/component/form/selectInput.templ

```diff
@@ -117,6 +120,49 @@ templ SelectInputWithLabelValuePair(
 	</div>
 }
 
+templ SelectInputWithLabelValuePairAndInputField(
```

Comment:
Why a select would have an input type?

Reply:
Because you can select an option that will have a specific value or type your own value in the input. It will make more sense by looking at this screenshot.
![image](https://github.com/user-attachments/assets/6489a295-ce07-498d-a6a1-d6155b5f4c44)